README.md

```diff
@@ -92,6 +94,25 @@ The following files are the central input files for ABACUS. Before executing the
 
 [back to top](#readme-top)
 
+## Run ABACUS
+
+After installation, the executable file named `abacus` can be found in the `build` folder. Copy this file and put it in the same folder as all required input files.
```

Comment:
The executable file is installed to a designated place. Users should not copy executable file. Please refer to [install.md](https://github.com/deepmodeling/abacus-develop/blob/develop/docs/install.md#configure).